.travis.yml

```diff
@@ -28,13 +28,11 @@ install:
   - conda info -a
 
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip coverage
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coveralls
   - source activate test-environment
-  - conda install numpy scipy matplotlib astropy scikit-learn h5py coveralls aipy six pycodestyle pydocstyle pytest-cov
-  - pip install git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata.git
-  - pip install git+https://github.com/HERA-Team/linsolve.git
-  - pip install git+https://github.com/HERA-Team/uvtools.git
-  - pip install git+https://github.com/HERA-Team/hera_cal.git
+  - pip install git+https://github.com/HERA-Team/linsolve.git   # only required until it updates
```

Comment:
@steven-murray what does this comment mean?

Reply:
It means that once the upstream PRs have been merged (eg linsolve and uvtools), these commands will not be necessary -- they can go straight into the `environment.yml`. I just wanted to remind ourselves of this so they don't stay there redundantly :-)

Reply:
ok, makes sense. But I wouldn't take from this comment that they need to be moved somewhere else as opposed to just removed.

Reply:
You're right, I just took them out. This PR isn't going to be merged until the upstream PRs are merged anyway.